eng/scripts/Invoke-GenerateAndBuildV2.ps1

```diff
@@ -108,6 +109,39 @@ if ($inputFileToGen) {
     UpdateExistingSDKByInputFiles -inputFilePaths $inputFileToGen -sdkRootPath $sdkPath -headSha $commitid -repoHttpsUrl $repoHttpsUrl -downloadUrlPrefix "$downloadUrlPrefix" -generatedSDKPackages $generatedSDKPackages
 }
 
+# generate sdk from cadl file
+if ($relatedCadlProjectFolder) {
+    for ($i = 0; $i -le $relatedCadlProjectFolder.Count - 1; $i++) {
+        $caldFolder = (Join-Path $swaggerDir $relatedCadlProjectFolder[$i]) -replace "\\", "/"
```

Comment:
You also misspelled `cadlFolder`. I recommend fixing the mistake so someone later doesn't spell it correctly, but break the script.

Reply:
Thanks @heaths updated